conf/application.conf

```diff
@@ -225,6 +225,12 @@ microservice {
       port = 9508
       enabled = true
     }
+
+    transit-movements-upscan-initiate {
```

Comment:
Or even just upscan ?

Reply:
No, the service we'll be connecting to is specifically called `upscan-initiate`. We should have this disambiguation in there in case we ever need to connect to another Upscan service, now or in the future.

Reply:
ok, we rename to `upscan-initiate`